### PR TITLE
How to install Mikrotik CHR on Hetzner Cloud

### PR DESCRIPTION
**Pull request by robert-hetzner**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.

### CODE REVIEW

File: tutorials/mikrotik-chr-basic-setup/01.en.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/mikrotik-chr-basic-setup"
+slug: "mikrotik-chr-basic-setup"
+date: "2021-02-19"
+title: "How to install Mikrotik CHR on Hetzner Cloud"
+short_description: "This tutorial is about a short guide on how to install Mikrotik CHR on Hetzner Cloud. This can be done in just a few steps."
```
Comment (asciiprod) [inline tutorials/mikrotik-chr-basic-setup/01.en.md:7]: This tutorial shows how to install Mikrotik CHR on Hetzner Cloud in just a few steps.

File: tutorials/mikrotik-chr-basic-setup/01.en.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+author_img: "https://avatars.githubusercontent.com/u/42843370?s=460&v=4"
+author_description: "Developer"
+language: "en"
+available_languages: ["en"]
+header_img: "header-3"
+---
+
+### Introduction
+
+We are focusing on installing the OS and doing a basic setup in this documentation. For any further details for configuration, look up in the official wiki from [MikroTik](https://wiki.mikrotik.com/wiki/Main_Page).
```
Comment (asciiprod) [inline tutorials/mikrotik-chr-basic-setup/01.en.md:20]: We focus on installing the OS and basic setup in this guide. For further configuration details you can refer to the official wiki from

File: tutorials/mikrotik-chr-basic-setup/01.en.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+If desired you can add an IP-Address to a user, from where it is only allowed to access.
+```
+# /user set <username> allowed-address=<IPv4>/<Netmask>
+```
+
+Now we want to disable all unnecessary services. Current running services can be shown via `# /ip service print`.  In this case we will disable all except `ssh` :
+```
+# /ip service disable telnet,ftp,www,api,api-ssl,winbox
+``` 
+We are recommending to change the `default ssh port 22` with any other desired.
```
Comment (asciiprod) [inline tutorials/mikrotik-chr-basic-setup/01.en.md:60]: We recommend

File: tutorials/mikrotik-chr-basic-setup/01.en.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+Afterwards you can reboot the server into the new installed OS. 
+Keep in mind, that the default login credentials are `user : admin` and `password : none`. Therefore we recommend on immediately disable the admin user and add a new one.
+Login to your server via ssh or the Hetzner Console and do the following commands:
+
+```
+# /user add name=<username> password=<userpassword> group=full
+# /user remove admin
+```
+If desired you can add an IP-Address to a user, from where it is only allowed to access.
```
Comment (asciiprod) [inline tutorials/mikrotik-chr-basic-setup/01.en.md:51]: Adding an IP address to a user will limit access to this user from this address only.

File: tutorials/mikrotik-chr-basic-setup/01.en.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: [submitter's name and email address here]
```
Comment (rook59) [inline tutorials/mikrotik-chr-basic-setup/01.en.md:146]: Please sign off here @robert-hetzner

**Review by rook59 — COMMENTED** [review]
Tutorial was written and tested internally, once signed off it's ready for final spelling check @leojakeson

**Pull request merged**